sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClientIntegrationTest.java

```diff
@@ -153,7 +153,7 @@ public void beginPurchaseandReleasePhoneNumbers(HttpClient httpClient) {
         named = "COMMUNICATION_SKIP_INT_PHONENUMBERS_TEST",
         matches = "(?i)(true)")
     @DisabledIfEnvironmentVariable(
-        named = "SKIP_LIVE_TEST",
+        named = "SKIP_UPDATE_CAPABILITIES_LIVE_TESTS",
```

Comment:
Doesn't the intention of the phone numbers pool is to avoid skipping the update capabilities test?

Reply:
Yes, but since the number of test jobs running in parallel is larger than the amount of phone numbers available in the pool (in the case of this SDK, 4 jobs vs 3 phone numbers), we cannot guarantee that all runners have a unique phone number assigned to them. Thus, for those runners that don't have a phone number assigned, the tests are skipped to avoid state conflicts.

Reply:
What prevents us from having 4 phone numbers in the pool?

Reply:
As discussed offline, we will go forward with only 3 phone numbers for the time being.